### PR TITLE
resolution of non-uniqueness in the intact file.

### PR DESCRIPTION
**Pull request by DSuveges**
* Non-uniqueness of the gene-id column in the pre-processed Intact file is sorted.
* Pre-processed Uniprot file is renamed from `uniprot_covid19_parsed.tsv` to `!uniprot_covid19_parsed.tsv` to ensure this is the first integrated dataset.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -242,7 +242,7 @@ $(ENSEMBLPARSED) $(UNIPROT2ENSEMBLDRAFT): $(ENSEMBL)
 	$(PIPENV) run python $(SRCDIR)/parsers/ensembl_parser.py -i $(ENSEMBL) -o $(ENSEMBLPARSED) -m $(UNIPROT2ENSEMBLDRAFT)
 
 $(INTACTCOVIDPARSED): $(INTACTCOVID) $(UNIPROTIDMAPPING) $(INTACTHUMAN)
-	$(PIPENV) run python $(SRCDIR)/parsers/intact_parser.py -i $(INTACTCOVID) -o $@ -m  $(UNIPROTIDMAPPING) -f $(INTACTHUMAN)
+	$(PIPENV) run python $(SRCDIR)/parsers/intact_parser.py -i $(INTACTCOVID) -o $@ -m  $(UNIPROT2ENSEMBL) -f $(INTACTHUMAN)
```
Comment (DSuveges) [inline Makefile:245]: The new map file is used to make sure the uniprot->ensembl mapping is as complete as possible.

File: src/parsers/Ensembl-Uniprot_map_generator.py [hunk 2]
```diff
@@ -27,9 +27,12 @@
 
     ## Report input data:
     print('[Info] Report on the Ensembl mapping file:')
+    print('[Info] Dropping non-Ensembl gene IDs...')
+    ensembl_map_df = ensembl_map_df.loc[ensembl_map_df.ensembl_id.str.match('ENSG.+')]
```
Comment (DSuveges) [inline src/parsers/Ensembl-Uniprot_map_generator.py:31]: The gene id-s in the ensembl derived map file are not `ENSG\d+` for 3880 proteins:
```
grep -v "ENSG"  temp/parsed_tables/uniprot2ensembl_draft.tsv |  head
```
Gives:
```
ensembl_id
LRG_376
LRG_376
LRG_376
LRG_571
LRG_571
LRG_571
LRG_777
LRG_777
LRG_656
```

Comment (AsierGonzalez) [reply src/parsers/Ensembl-Uniprot_map_generator.py:31]: Why are we dropping these ids? I agree that they are odd but is the decision to drop them justified?

Comment (AsierGonzalez) [reply src/parsers/Ensembl-Uniprot_map_generator.py:31]: The UniProt mapping of some of those ids clash with Ensembl ids (thanks @DSuveges):
```
H3BQV5	ENSG00000197943
H3BQV5	LRG_376
```
And they don't seem to be included in the Open Targets platform, so it makes sense to remove them.

**Review by AsierGonzalez — APPROVED** [review]

**Pull request merged**